src/country_data.js

```diff
@@ -1269,6 +1269,8 @@ for (var i = 0; i < allCountries.length; i++) {
     // format
     if (c[3]) {
       allCountries[i].format = c[3];
+    } else {
+      allCountries[i].format = "+" + Array(allCountries[i].dialCode.length + 1).join(".").toString() + " .";
```

Comment:
Maybe it's my unfamiliarity with this codebase, but I feel  like this could use a comment to explain what is going on.